pkg/lib/session/manager.go

```diff
@@ -138,7 +144,16 @@ func (m *Manager) Logout(session Session, rw http.ResponseWriter) error {
 }
 
 func (m *Manager) Revoke(session Session, isAdminAPI bool) error {
-	_, err := m.invalidate(session, isAdminAPI)
+	_, err := m.invalidate(session, &signedOutEventOption{IsAdminAPI: isAdminAPI})
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (m *Manager) Delete(session Session) error {
```

Comment:
It is better to include NoEvent and WithEvent in the method to tell the difference. 

```golang
RevokeWithEvent(session Session, isAdminAPI bool) error
RevokeNoEvent(session Session, isAdminAPI bool) error
```